winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Value;
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+workBook.ActiveSheet.Columns[2].NumberFormat = "$0.0";
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-column-formatting.png" alt="Formatting the Column while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+### Alternate row styling without using CellsExportHandler
+
+In the following code sample, the background color of rows in excel is changed based on the row index using conditional formatting for better performance.
+
+Reference:
+[Conditional Formatting](http://help.syncfusion.com/file-formats/xlsio/working-with-conditional-formatting)
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Value;
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+IConditionalFormats condition = workBook.ActiveSheet.Range[2, 1, this.treeGrid.View.Nodes.Count + 1, this.treeGrid.Columns.Count].ConditionalFormats;
+IConditionalFormat condition1 = condition.AddCondition();
+condition1.FormatType = ExcelCFType.Formula;
+condition1.FirstFormula = "MOD(ROW(),2)=0";
+condition1.BackColorRGB = Syncfusion.Drawing.Color.Pink;
+IConditionalFormat condition2 = condition.AddCondition();
+condition2.FormatType = ExcelCFType.Formula;
+condition2.FirstFormula = "MOD(ROW(),2)=1";
+condition2.BackColorRGB = Syncfusion.Drawing.Color.LightGray;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-changing-row-style.png" alt="Displaying Row Style while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+## How to
+
+### Export multiple TreeGrids to single Excel sheet
+
+You can export multiple TreeGrids to single Excel sheet by merging one TreeGrid worksheet into another using the `Worksheet.UsedRange.CopyTo` method.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook1 = excelEngine.Excel.Workbooks[0];
+var worksheet1 = workBook1.Worksheets[0];
+            
+excelEngine = treeGrid1.ExportToExcel(options);
+var workBook2 = excelEngine.Excel.Workbooks[0];
+var worksheet2 = workBook2.Worksheets[0];
+
+var columnCount = this.treeGrid1.Columns.Count;
+//Merge the One TreeGrid WorkSheet into the other TreeGrid WorkSheet

```

Comment:
//Merge One TreeGrid WorkSheet into the other TreeGrid WorkSheet.

Reply:
Updated